setup.py

```diff
@@ -63,10 +64,7 @@ def run(self):
         'Development Status :: 4 - Beta',
         'Intended Audience :: Developers',
         'License :: OSI Approved :: MIT License',
-        'Programming Language :: Python :: 2',
-        'Programming Language :: Python :: 2.7',
         'Programming Language :: Python :: 3',
-        'Programming Language :: Python :: 3.3',
         'Programming Language :: Python :: 3.4',
```

Comment:
Seems like this should get deleted as well since we are only testing Python 3.5+